core/src/main/java/org/opentosca/toscana/core/data/LogEntry.java

```diff
@@ -0,0 +1,16 @@
+package org.opentosca.toscana.core.data;
+
+public class LogEntry {
+
+    public final long TIMESTAMP;
```

Comment:
I think this should be lower case letters.

All Capital Letters are usualy used for static variables/constants.

Reply:
Your right. I was unsure about naming conventions regarding final but non-static fields.
WIll change this